src/main/java/bio/terra/metadata/StudyTableColumn.java

```diff
@@ -1,39 +1,27 @@
 package bio.terra.metadata;
 
-import bio.terra.model.ColumnModel;
-
 import java.util.UUID;
 
 public class StudyTableColumn {
     private String name;
     private String type;
     private UUID id;
-
-    public StudyTableColumn(String name, String type) {
-        this.name = name;
-        this.type = type;
-    }
-
-    public StudyTableColumn(ColumnModel columnModel) {
-        this.name = columnModel.getName();
-        this.type = columnModel.getDatatype();
-    }
+    private volatile StudyTable inTable;
```

Comment:
Why volatile? You probably have a good reason; maybe add a comment explaining?

Reply:
ah... i meant transient and not volatile. but we should definitely discuss this. volatile (if i remember correctly) is a key to the compiler not to optimize this value. basically don't assume you know when the value can change. this is typically used when the value can be updated in another thread. transient means do not serialize this value. since this is a pointer back to the table that references it, i think if it wasn't transient and we tried to serialize this object, we might get an infinite serialization. if we know this object will not be part of stairway context that is serialized, we probably don't need it.